automated-testing/ui-testing/readme.md

```diff
@@ -0,0 +1,64 @@
+# UI Testing
+
+This section is primarily geared towards web-based UIs, but the guidance is similar for mobile and OS based applications.  
```

Comment:
nit - I know these are common acronyms, but how about we spill then out the first time?

Reply:
Good catch, updating.